contents/tutorials/nextjs-error-monitoring.md

```diff
@@ -13,7 +13,7 @@ To help you set this up, this tutorial details how to create a basic Next.js app
 
 ## 1. Creating a Next.js app
 
-Start by ensuring [Node.js is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (version 18.0 or newer) then run the following command. Say **no** to TypeScript, **yes** to app router, and the defaults for other options.
+Start by ensuring [Node.js is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (version 18.0 or newer) then run the following command. Say **no** to TypeScript, **yes** to app router, and the defaults for other options.
```

Comment:
lol why are the two types of spaces different 😢

Reply:
🤷‍♂️